Library/Formula/minised.rb

```diff
@@ -1,12 +1,15 @@
-require 'formula'
-
 class Minised < Formula
-  homepage 'http://www.exactcode.de/site/open_source/minised/'
-  url 'http://dl.exactcode.de/oss/minised/minised-1.14.tar.gz'
-  sha1 'ddfd770b911ae482fd79a6d8f5f1e731ad2784e8'
+  homepage "http://www.exactcode.de/site/open_source/minised/"
+  url "http://dl.exactcode.de/oss/minised/minised-1.15.tar.gz"
+  sha256 "ada36a55b71d1f2eb61f2f3b95f112708ce51e69f601bf5ea5d7acb7c21b3481"
 
   def install
     system "make"
     system "make", "DESTDIR=#{prefix}", "PREFIX=", "install"
   end
+
+  test do
+    output = `echo 'hello world' | #{bin}/minised 's:o::'`
```

Comment:
For style points:
`output = pipe_output("#{bin}/minised 's:o::'", "hello world", 0)`